lib/db/mysql.js

```diff
@@ -540,6 +540,63 @@ module.exports = function (log, error) {
     )
   }
 
+  // VERIFICATION REMINDERS
+
+  // INSERT : id, uid, email, emailCode, type, acceptLanguage, createdAt
+  var CREATE_REMINDER = 'CALL createVerificationReminder_1(?, ?, ?, ?)'
+
+  MySql.prototype.createVerificationReminder = function (request) {
```

Comment:
Do you think it might be better to pass in the `uid` and `type` instead of `request`?